services/wfm/src/main/java/org/openkilda/wfm/topology/utils/KibanaLogManager.java

```diff
@@ -0,0 +1,76 @@
+/* Copyright 2018 Telstra Open Source
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ */
+
+package org.openkilda.wfm.topology.utils;
+
+import static org.apache.log4j.Level.DEBUG;
+import static org.apache.log4j.Level.ERROR;
+import static org.apache.log4j.Level.INFO;
+import static org.apache.log4j.Level.WARN;
+
+import org.apache.log4j.Level;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.slf4j.MDC;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+import javax.annotation.Nullable;
+
+public class KibanaLogManager {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(KibanaLogManager.class);
+
+    public void debug(String message, @Nullable Map<Object, Object> data) {
```

Comment:
em... what is this?

This "manager" must provide methods like "islDiscovery(SwitchId srcSwitch, int srcPort, SwitchId dstSwitch, int dstPort)". External world must know nothing about produced messages, all "logging" logic must be incapsulated inside `KibanaLogManager`.